### PR TITLE
Feat: Display Fuel Supply in Report Details Section - 717

### LINKED ISSUE #717: LCFS - Fuel supply, display on report detail section
**Description:** After a BCeID user has entered data into the Fuel supply input form and has returned to the main report a new section titled 'Report details' will be visible directly under the bulleted link list. The Fuel supply display (AG Grid) table will be contained in its own accordion and open by default.  (If other data had already been added then the Report details section would have already existed.) There is also a new section titled 'Summary & declaration' but this will be future development. A placeholder could be included until the Summary work is done later.

- Because the input form has autosave, it doesn't matter if the user clicked the 'Save & return to report' button or navigated back to the report using a different method. The data will be displayed on the report.
- The AG Grid table will be limited to displaying 10 rows of data with paging. If the table has less than 10 rows don't show the paging.
- Horizontal scrolling, if necessary, will be self contained within the accordion.
- There is a pencil icon that will return the user to the input form to make further edits and additions. (There is no changelog to these types of edits). Similarly, the bulleted text link to the Fuel supply input page will return the user to the input form with their data being displayed in edit mode.

**Wireframe:** 
The table is within an accordion that is open by default, with a pencil icon for edits.
https://preview.uxpin.com/da68049c0af15edd7ce4e47e4bee431fed40c30d#/pages/178710561?mode=m

**Purpose and benefit to user:** This is the main interaction for the user between adding Fuel supply data to the report and seeing the report being built out.

**Acceptance Criteria:**

Given I am a BCeID user, when I:
- have entered data into the Fuel supply input form and returned to the main report, then a new section titled 'Report details' will be visible directly under the bulleted link list.
- the Fuel supply display (AG Grid) table, then it will be contained in its own accordion and open by default. If other data had already been added, then the 'Report details' section would already exist.
- navigate to the main report, then there will also be a new section titled 'Summary & declaration' as a placeholder for future development.
- use the autosave feature, then my data will be displayed on the report regardless of whether I clicked the 'Save & return to report' button or navigated back to the report using a different method.
- view the AG Grid table, then it will display up to 10 rows of data with paging enabled if there are more than 10 rows, and if there are fewer than 10 rows, paging will not be shown.
- need to scroll horizontally, then it will be self-contained within the accordion.
- click the pencil icon, then I will be returned to the input form to make further edits and additions, and no changelog will be maintained for these types of edits.
- click the bulleted text link to the Fuel supply input page, then I will be returned to the input form with my data being displayed in edit mode.

**Development Checklist:**

- A. Implement the 'Report details' section to appear under the bulleted link list when data is entered into the Fuel supply input form.
- B. Ensure the Fuel supply display (AG Grid) table is within its own accordion, open by default, and supports paging for more than 10 rows of data.
- C. Add a placeholder section titled 'Summary & declaration' for future development.
- D. Enable autosave functionality to ensure data is displayed on the report regardless of navigation method.
- E. Contain horizontal scrolling within the accordion for the AG Grid table.
- F. Add a pencil icon to allow users to return to the input form for edits and additions without a changelog.
- G. Ensure the bulleted text link to the Fuel supply input page returns users to the input form with data in edit mode.

**Notes:**

### PR DESCRIPTION
**Pull request by hamed-valiollahi**
This PR adds the Fuel supply display table to the 'Report details' section. A few table column values need to be determined.

### Improvements
- Add Docker Compose container and volume names to avoid creating numerous node module volumes over time and to optimize storage usage.

Closes #717

### CODE REVIEW

**Review by AlexZorkin — CHANGES_REQUESTED** [review]
Looks good. Just two minor comments. ty

File: backend/lcfs/web/api/fuel_supply/schema.py [hunk 1]
```diff
@@ -71,13 +71,13 @@ class FuelTypeSchema(BaseSchema):
...
     fossil_derived: bool
     default_carbon_intensity: float
-    unit: str
-    energy_density: Optional[EnergyDensitySchema]
-    provisions: List[ProvisionOfTheActSchema]
-    fuel_categories: List[FuelCategorySchema]
-    eer_ratios: List[EnergyEffectivenessRatioSchema]
-    target_carbon_intensities: List[TargetCarbonIntensitySchema]
-    fuel_codes: Optional[List[FuelCodeSchema]] = []
+    # unit: str
```
[the remark this answers is not in the document]
Comment (hamed-valiollahi) [reply backend/lcfs/web/api/fuel_supply/schema.py]: Of course, I removed all these unneeded commanded lines.

File: docker-compose.yml
[the remark this answers is not in the document]
Comment (AlexZorkin) [reply docker-compose.yml:1]: Yes thats great thank you! remove all those unnecessary volumes.

[the remark this answers is not in the document]
Comment (hamed-valiollahi) [reply docker-compose.yml:1]: Sure, thank you for your feedback, Alex!

**Comment by prv-proton** [comment]
closing as already merged with editor ticket

**Pull request closed without merging**